front/cypress/e2e/report_builder/idea_template.cy.ts

```diff
@@ -105,6 +105,60 @@ describe('Idea template', () => {
       // Ensure we're back to the empty state
       cy.get('#e2e-create-report-button').should('exist');
     });
+
+    it('creates a report from a template and allows editing it', () => {
+      cy.setAdminLoginCookie();
+      cy.apiCreateReportBuilder().then((report) => {
+        const reportId = report.body.data.id;
+        cy.visit(
+          `/admin/reporting/report-builder/${reportId}/editor?templateProjectId=${projectId}`
+        );
+
+        cy.wait(1000);
```

Comment:
Two tests (this and global_report.cy.ts) seem to be very similar, but still with some differences. E.g., this one uses `wait` and `randomString`. Why do we need these differences?

Reply:
The `randomString` is just a more guaranteed way of making sure that the string is unique and I am checking the correct thing. We could use `randomString` in all places I guess. But not very important IMO